src/kaboom.ts

```diff
@@ -1006,35 +1080,7 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 	const game = {
 
 		// general events
-		ev: new EventHandler<{
-			mouseMove: [],
-			mouseDown: [MouseButton],
-			mousePress: [MouseButton],
-			mouseRelease: [MouseButton],
-			charInput: [string],
-			keyPress: [Key],
-			keyDown: [Key],
-			keyPressRepeat: [Key],
-			keyRelease: [Key],
-			touchStart: [Vec2, Touch],
-			touchMove: [Vec2, Touch],
-			touchEnd: [Vec2, Touch],
-			virtualButtonDown: [VirtualButton],
-			virtualButtonPress: [VirtualButton],
-			virtualButtonRelease: [VirtualButton],
-			gamepadButtonDown: [string],
-			gamepadButtonPress: [string],
-			gamepadButtonRelease: [string],
-			scroll: [Vec2],
-			add: [GameObj],
-			destroy: [GameObj],
-			load: [],
-			loadUpdate: [number],
-			error: [Error],
-			input: [],
-			frameEnd: [],
-			resize: [number, number, number, number],
-		}>(),
+		ev: new EventHandler(),
```

Comment:
Recently added the type info for all events, like
```js
eventName: [eventArgs]
```
can you keep the type defs and add ones for the gamepad events?